uk_bin_collection/uk_bin_collection/councils/DurhamCouncil.py

```
@@ -1,49 +1,87 @@
+    # `page` arg. input.json sets skip_get_url=true to avoid a wasted GET.
     def parse_data(self, page: str, **kwargs) -> dict:
-        url = "https://www.durham.gov.uk/bincollections?uprn="
         uprn = kwargs.get("uprn")
         check_uprn(uprn)
-        url += uprn
-        requests.packages.urllib3.disable_warnings()
-        page = requests.get(url)
 
-        # Make a BS4 object
-        soup = BeautifulSoup(page.text, features="html.parser")
+        response = requests.post(
```

Ruff (0.15.13)

[error] Probable use of `requests` call without timeout

(S113)